src/main/webapp/src/app/my_gardens_page/my-gardens.component.css

```diff
@@ -0,0 +1,48 @@
+.avatar {
+  background-size: cover;
+  height: 50px;
+  padding-top: 5px;
+}
+.avatar-image {
+  width: 50px;
+}
+
+.button {
```

Comment:
Since it seems like we want to keep the same style for buttons, we should move this styling to the common folder. I  would say these styling changes should be made in a different PR so it can be changed in all files evenly.